pages/common/license.md

```diff
@@ -0,0 +1,19 @@
+# license
+
+> Create license files for open-source projects.
+
+- Create a license:
```

Comment:
@igorshubovych I would have expected that, but the description of the example following this suggests otherwise. So it's best to be explicit, IMO.